managed/src/main/resources/db/migration/default/common/V64__Alter_Customer_Config_Table.sql

```diff
@@ -0,0 +1,4 @@
+-- Copyright (c) YugaByte, Inc.
+
+alter table if exists customer_config add column if not exists config_name varchar(64) null;
+update if exists customer_config set config_name = concat(name, '-Default') where config_name is null;
```

Comment:
Looks good. Please check that it works both for H2 (junit tests) and PostgreSQL.

Reply:
guess `update if exists customer_config` is incorrect . Will update this query.

Reply:
@SergeyPotachev `if exists ` is required in the second statement? Shall I safely assume that the table would exist and set the value? ( I don't see most of the migrations using the same check )